gtsam/inference/Factor.h

```diff
@@ -128,6 +131,13 @@ typedef FastSet<FactorIndex> FactorIndexSet;
     */
     size_t size() const { return keys_.size(); }
 
+    /// See NonLinearFactor::active().
+    bool active() const { return active_; }
+
+    /// Changes the default "active" status of this factor.
+    void active(bool newActiveValue) { active_ = newActiveValue; }
```

Comment:
I know some code bases like this style, but we have not done this in GTSAM. Prefer `setActive` or have `activate` and `deactivate` methods. That being said, I don't like imperative change, at all.